README.md

```diff
@@ -1,57 +1,25 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
-
-## Getting Started
-
-### Prerequisites
-
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
+# Webapp with a cosmos Db
+ 
+ A sample webapp deployed on app service with cosmos db as backend. It counts the number of visitors visiting the page and inserts the same to a sample collection on cosmos db.
```

Comment:
WebApp with **a** Cosmos DB
with cosmos db as **database**
**into** a sample collection **in** Cosmos DB